csharp/src/Google.Protobuf.Test/Buffers/ArrayBufferWriter.cs

```diff
@@ -42,18 +42,18 @@ namespace Google.Protobuf.Buffers
     /// ArrayBufferWriter is originally from corefx, and has been contributed to Protobuf
     /// https://github.com/dotnet/runtime/blob/071da4c41aa808c949a773b92dca6f88de9d11f3/src/libraries/Common/src/System/Buffers/ArrayBufferWriter.cs
     /// </summary>
-    internal sealed class ArrayBufferWriter<T> : IBufferWriter<T>
+    internal sealed class TestArrayBufferWriter<T> : IBufferWriter<T>
```

Comment:
Renamed to avoid conflict with ArrayBufferWriter in .NET 5. It isn't completely removed because we've customized it with max grow by logic for testing.